src/awscdk/base.ts

```diff
@@ -78,6 +78,14 @@ export interface CDKPipelineOptions {
    */
   readonly stackPrefix?: string;
 
+  /**
+   * If set to true all CDK actions will also include <stackName>/* to deploy/diff/destroy substacks of the main stack.
```

Comment:
why the concept of / to identify sub-stacks, is that common or mandated ?

Reply:
That happens when you create a new Stack inside another stack instance

```
class MyStack extends Stack {
  constructor(...) {
    super(...)

    new Stack(this, 'something', ...)
  }
}
```

Reply:
Thanks, I didnt know that.